pkg/BUILD.bazel

```diff
@@ -3383,8 +3383,8 @@ test_suite(
     name = "ccl_tests",
     tags = [
         "-broken_in_bazel",
-        "ccl_test",
         "-integration",
+        "ccl_test",
```

Comment:
Interesting...

Reply:
I don't understand this... just keeps insisting on rearranging it....

Reply:
Just running ./dev/lint